spec/requests/forms/what_happens_next_controller_spec.rb

```diff
@@ -66,7 +72,7 @@
     end
 
     context "when the user is not authorised to view the form" do
-      let(:form_organisation_id) { 999 }
+      let(:user) { build :user }
```

Comment:
These tests have been changed so that the form is in the same group, but we try to access it with another user who  isn't a member.

This is easier to change than making the test setup conditional.